src/cmd/troubleshoot/context.go

```diff
@@ -0,0 +1,41 @@
+package troubleshoot
+
+import (
+	"net/http"
+	"net/url"
+
+	"github.com/Dynatrace/dynatrace-operator/src/api/v1beta1"
+	corev1 "k8s.io/api/core/v1"
+	"sigs.k8s.io/controller-runtime/pkg/client"
+)
+
+type troubleshootContext struct {
```

Comment:
can we pass in the actual `context.Context` here, so it can be used instead of `context.TODO()`?